Piwik.Tracker/PiwikTracker.cs

```diff
@@ -182,7 +182,7 @@ public class PiwikTracker
         private bool _configCookiesDisabled;
         private string _configCookiePath = DefaultCookiePath;
         private string _configCookieDomain = "";
-        private readonly long _currentTs = (long)(DateTime.Now - new DateTime(1970, 1, 1)).TotalSeconds;
+        private readonly long _currentTs = (long)(DateTime.UtcNow - new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc)).TotalSeconds;

```

Comment:
For consistency, I would initialize Unix epoch as `new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc)` always, though it doesn't make any difference in this instance.